angularSTEP/src/app/signup/signup.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="example-container">
```

Comment:
Can "example-container" be changed to something more appropriate for this view? Please apply this to other placeholder values I missed in review as well!

Reply:
Oops! Yes, of course. Just totally missed that I still had that in there...